<launch>

<arg name="color_config_file" default="$(find color_detector)/config/colors.json" />
<arg name="color" default="" />

<!--<include file="$(find openni_launch)/launch/openni.launch"/>-->
<node name="color_detector" pkg="color_detector" type="color_detector">
	<param name="color_string" value="$(arg color)"/>
	<param name="color_config_path" value="$(arg color_config_file)"/>
</node>

</launch>